src/main/java/org/spongepowered/api/command/CommandManager.java

```diff
@@ -101,6 +102,50 @@
     Optional<CommandMapping> register(Object plugin, CommandCallable callable, List<String> aliases, Function<List<String>, List<String>> callback);
 
     /**
+     * Installs a {@link AbstractCommandModule} into this command manager.
+     *
+     * @param plugin The plugin instance
+     * @param module The command module
+     */
+    void install(Object plugin, AbstractCommandModule module);
```

Comment:
Is CommandManager only ever expected to be implemented by Sponge? if not, you might need a separate interface for CommandManagers that can accept an AbstractCommandModule.